xbmc/cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecFFmpeg.cpp

```diff
@@ -247,6 +247,15 @@ enum AVPixelFormat CDVDVideoCodecFFmpeg::GetFormat( struct AVCodecContext * avct
       {
         ctx->m_processInfo.SetVideoPixelFormat(pixFmtName ? pixFmtName : "");
         ctx->SetHardware(dec);
+
+        std::list<EINTERLACEMETHOD> deintMethods;
```

Comment:
ah, this is for mmal. then it should be done in mmal. the decoder has the processInfo instance.